roles/contiv/tasks/ovs.yml

```diff
@@ -0,0 +1,24 @@
+---
+- include: packageManagerInstall.yml
+  when: source_type == "packageManager"
+  tags:
+    - binary-update
+
+- name: OVS | Configure selinux for ovs
+  shell: semanage permissive -a openvswitch_t
```

Comment:
At the moment this is necessary as long as we install ovs separate from Openshift. Without this the contiv data plane does not work correctly. When we switch to using the ovs that is already installed by openswitch (see the other comment on this), at that point we can decide whether to retain this or not. However for now, this is necessary for correct functional operation of Contiv.